coap/examples/coap_server/main/idf_component.yml

```diff
@@ -2,5 +2,5 @@ version: "1.0.0"
 description: CoAP Server Example
 dependencies:
   espressif/coap:
-    version: "^4.3.0"
+    version: "^4.3.2"
```

Comment:
This change can be reverted. The rule `^` (>=4.3.0, <=5.0.0) shall allow to pick latest minor version of the component automatically.

Reply:
Done

Reply:
Also reverted coap/examples/coap_client/main/idf_component.yml